examples/README.md

```diff
@@ -106,146 +109,139 @@ the wrong directory.
+    curl -k -u "admin:hardpass" "https://192.168.242.55:8080/v2/apps/hello-world" | python -m json.tool
 
 That should give you a nicely formated output of the current state of the app.
 
-Notice that we didn't need headers `-h` and the http verb was the default GET so we didn't need `-X`. We
-still needed the -k to get around our self signed ssl certificate.
+Notice that we didn't need headers `-h` and the http verb was the default GET,
+so we didn't need `-X`. We still needed the -k to get around our self signed ssl
+certificate.
 
 ##### Delete App
 
 From this REST API, the call to destroy the application is:
 
-    curl -k  "https://admin:hardpass@192.168.242.55:8080/v2/apps/hello-world" -X DELETE
+    curl -k -X DELETE -u "admin:hardpass" "https://192.168.242.55:8080/v2/apps/hello-world"
 
-which is the same as the current status path but with the HTTP DELETE verb instead of the default GET.  If successful you
-will get something like:
+which is the same as the current status path but with the HTTP DELETE verb
+instead of the default GET. If the call is successful you will get something
+like:
 
     {"version":"2015-12-14T06:44:37.378Z","deploymentId":"e7680e8e-d073-4c57-9f64-e73d8b634398"}
 
-you could check the status again, per the previous section. It should give you `{"message":"App '/hello-world' does not exist"}`
-
+Now you can check the status again, as per the previous section. It should give
+you `{"message":"App '/hello-world' does not exist"}`.
 
-## Where is my Application
 
-In version 0.5.0 the vagrant build does not have traefik in it and so service discovery is a bit convoluted but not too
-bad.
+## Where is my Application?
 
-In a browser, go to the IP of your vagrant cluster. This will be the value in your Vagrantfile as described earlier.
+In a browser, go to the IP of one of your vagrant control nodes. (Some browsers,
+like Firefox, will outright reject the SSL certificate, so you may need to use
+another for this step). This will be the value in your Vagrantfile as described
+earlier.
 
-![mantlui 192.168.242.55/ui](./images/mantlui.png)
+![mantlui 192.168.100.101/ui](./images/mantlui.png)
 
-Choose the Marathon "web UI" button and you should see:
+Choose the Marathon "Web UI" button and you should see something like this:
 
 ![marathonui](./images/marathonui.png)
 
-Click on your application, hello-world (Note: if its not there. you probably deleted it working through the steps above.
-Just start it again. ) and you should see:
+Click on your application, hello-world (if its not there, you probably deleted
+it while working through the steps above, but you can just start it again.)
+and you should see:
 
 ![marathonui at application](./images/marathonapp.png)
 
-You'll notice that there are two instances.  Each one has a line under it in gray `default:<####>` where <####> is some
-port number. In the picture above, ports 9061 and 25312.
+You'll notice that there are two instances. Each one has a line under it in
```

Comment:
fyi, on my default vagrant install, there wasn't enough memory to run both instances.